.ci/releaseBuild.yml

```diff
@@ -97,11 +97,26 @@ stages:
           **\*.psm1
         useMinimatch: true
 
-    # finally publish the parts of the build which will be used in the next stages
-    # if it's not published, the subsequent stages will not be able to access it.
-    - publish: "$(Build.SourcesDirectory)/OSS_Microsoft_CompatPowerShellGet"
-      artifact: build
-      displayName: publish build directory
+    # Register local folder as PSRepository
+    # Then publish-module to that folder 
+    - pwsh: |
+        $localRepo = "$(Build.SourcesDirectory)/local"  
+        New-Item -Path $localRepo -ItemType Directory -Force
+        $vstsCommandString = "vso[task.setvariable variable=localRepo]${localRepo}"
+        Write-Host "local repo is: " + $vstsCommandString
+        Write-Host "##$vstsCommandString"
+
+        Register-PSRepository -Name local -SourceLocation $(localRepo)
+        Publish-Module -Path $(signOutPath) -Repository local

```

Comment:
Should upload the nupkg file to pipeline artifacts so it can be downloaded in the publish stage for pushing to powershellgallery.com